iron-form.html

```diff
@@ -162,6 +171,21 @@
       this.fire('iron-form-submit', json);
     },
 
+    /**
+     * Resets all non-disabled elements to their initial values.
+     */
+    reset: function() {
+      // Trigger the native form validation, which in turn will also call
+      // `_resetCustomElements` to reset the form's custom element. Doing this
+      // prevents the event from being triggered twice.
+      HTMLFormElement.prototype.reset.call(this);
```

Comment:
How is this different from what happens without this override? I'm just curious, since at a glance it looks like the method is being overridden by implementation that just calls the overridden method anyway (and does nothing else).